.github/workflows/ci.yml

```diff
@@ -191,13 +191,42 @@ jobs:
         run: ./gradlew muzzle --init-script ../../.github/scripts/local.init.gradle.kts
         working-directory: examples/extension
 
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+
+      - name: Check markdown links
+        run: |
+          npm install -g markdown-link-check
+          find . -type f \
+                 -name '*.md' \
+                 -not -path './.github/*' \
+                 -not -path './node_modules/*' \
+                 -print0 \
+            | xargs -0 -n1 markdown-link-check --config .github/scripts/markdown_link_check_config.json
+
+  markdown-misspell-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+
+      - name: Check markdown for common misspellings
+        run: |
+          curl -L -o ./install-misspell.sh https://git.io/misspell
+          sh ./install-misspell.sh
+          ./bin/misspell -error ./**/*.md
```

Comment:
I think spell checking on Java code could be nice, not sure if it will work well if not restricted to only Java comments though.

Reply:
great idea, found several more, definitely makes the tool more worth it